### PR TITLE
IDMP-366: Add RIM related IDMP concept: Packaged Medicinal Product

### PR DESCRIPTION
**Pull request by ElisaKendall**
Description: Added packaged medicinal product, including related concepts for container, package item, and other containers, as well as batch and batch identifier

Signed-off-by: Elisa Kendall <ekendall@thematix.com>

### CODE REVIEW

**Review by Heiner-Oberkampf — DISMISSED** [review]
Quick review. seems good

**Review by mereolog — CHANGES_REQUESTED** [review]

File: ISO/ISO11615-MedicinalProducts.rdf [hunk 1]
```diff
@@ -721,8 +828,23 @@
...
+				<owl:onProperty rdf:resource="&cmns-col;isPartOf"/>
+				<owl:someValuesFrom rdf:resource="&idmp-mprd;MedicinalProduct"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>medicinal product container</rdfs:label>
+		<skos:definition>medicinal product container that is part of a medicinal product and is used for storage, identification and/or transport of the components of the medicinal product</skos:definition>
+		<idmp-cmpl:hasConformanceToISOLevel rdf:resource="&idmp-cmpl;DefinitionallyConformant"/>
+		<cmns-av:adaptedFrom>ISO 11615:2017 Health informatics - Identification of medicinal products (IDMP) - Data elements and structures for the unique identification and exchange of regulated information on substances, clause 3.1.16</cmns-av:adaptedFrom>
+	</owl:Class>
+
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply ISO/ISO11615-MedicinalProducts.rdf:844]: @merelog They are defined that way in the IDMP standards. In some cases the definition talks about a container that touches the pharmaceutical product (immediate container), in others the representation is for packaging that is in the "box" but doesn't touch the pharmaceutical product (intermediate container) and there is also outer packaging (the box). Of course, some drugs such as a bottle of tablets may not have all of these containers, but there are cases where all of them are present. And then there is this notion of a package item - which is the box or bottle itself, which is also manufactured.

File: ISO/ISO11615-MedicinalProducts.rdf [hunk 2]
```diff
@@ -784,6 +906,55 @@
...
+		<rdfs:subClassOf rdf:resource="&idmp-mprd;MedicinalProductContainer"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&idmp-mprd;contains"/>
+				<owl:someValuesFrom rdf:resource="&idmp-mprd;MedicinalProduct"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>outer packaging</rdfs:label>
+		<dct:source>ISO 11615:2017 Health informatics - Identification of medicinal products (IDMP) - Data elements and structures for the unique identification and exchange of regulated information on substances, clause 3.1.57</dct:source>
+		<skos:definition>external container in which a Medicinal Product is supplied</skos:definition>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply ISO/ISO11615-MedicinalProducts.rdf:920]: @merelog I didn't catch that one and fix it (copy/paste from the standard) - well spotted. I'll fix it.

File: ISO/ISO11615-MedicinalProducts.rdf [hunk 3]
```diff
@@ -358,6 +345,34 @@
 		<skos:definition>party that delegates some role, authority, capability, or control to another party</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&idmp-mprd;Batch">
+		<rdfs:subClassOf rdf:resource="&cmns-prd;Product"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&cmns-col;comprises"/>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply ISO/ISO11615-MedicinalProducts.rdf:352]: @mereolog I need to check the logic - contains is transitive and comprises is not, so I had to eliminate some min 0 restrictions as a consequence, which is why the allValuesFrom above is currently linked to owl:Thing. I might be able to change that to manufactured item but I need to walk the logic again to see what will happen if I do that.

File: ISO/ISO11615-MedicinalProducts.rdf (hunk 2 above)
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply ISO/ISO11615-MedicinalProducts.rdf:920]: @mereolog Done.

**Review by mereolog — APPROVED** [review]

File: EXT/Examples/EuropeanUnionClinicalTrialsRegister.rdf [hunk 4]
```diff
@@ -256,12 +259,12 @@
 	
 	<owl:NamedIndividual rdf:about="&idmp-euctr;GlaxoSmithKlineBiologicalsAsMarketingAuthorizationHolder">
```
Comment (Heiner-Oberkampf) [inline EXT/Examples/EuropeanUnionClinicalTrialsRegister.rdf:260]: These URIs are OK for specifying examples, but in general I would not see this in the IDMP Ontology itself for organizations as they are mastered in SPOR or LEI or some other registry - not by us.

**Review by Heiner-Oberkampf — APPROVED** [review]

**Pull request merged**